$MirOS: ports/devel/glib/patches/patch-glib_gunibreak_h,v 1.1 2008/03/06 22:08:24 tg Exp $

	Support code for U+1E9E

--- glib/gunibreak.h.orig	2009-04-01 01:04:20.000000000 +0200
+++ glib/gunibreak.h	2009-07-14 20:28:31.000000000 +0200
@@ -13877,7 +13877,7 @@ static const gint8 break_property_data[]
     G_UNICODE_BREAK_ALPHABETIC, G_UNICODE_BREAK_ALPHABETIC, 
     G_UNICODE_BREAK_ALPHABETIC, G_UNICODE_BREAK_ALPHABETIC, 
     G_UNICODE_BREAK_UNKNOWN, G_UNICODE_BREAK_UNKNOWN, 
-    G_UNICODE_BREAK_UNKNOWN, G_UNICODE_BREAK_UNKNOWN, 
+    G_UNICODE_BREAK_ALPHABETIC, G_UNICODE_BREAK_UNKNOWN, 
     G_UNICODE_BREAK_ALPHABETIC, G_UNICODE_BREAK_ALPHABETIC, 
     G_UNICODE_BREAK_ALPHABETIC, G_UNICODE_BREAK_ALPHABETIC, 
     G_UNICODE_BREAK_ALPHABETIC, G_UNICODE_BREAK_ALPHABETIC, 
